src/components/styles/menu.less

```diff
@@ -0,0 +1,81 @@
+@import './constants.less';
+
+.reactist_menulist[role='menu'] {
+    display: block;
+    white-space: nowrap;
+    background: hsla(0, 100%, 100%, 0.99);
+    outline: none;
+    font-size: 85%;
```

Comment:
Shouldn't we rely on our font size constants (e.g. `@smaller_font_size`) here and let users override if necessary?